__tests__/app.test.js

```diff
@@ -0,0 +1,63 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe(`/api/topics tests`, () => {
+  describe(`GET tests`, () => {
+    test(`/api/topics, returns an array of objects`, () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          topics.forEach((topic) =>
+            expect(topic).toEqual(
+              expect.objectContaining({
+                slug: expect.any(String),
+                description: expect.any(String),
+              })
+            )
```

Comment:
Could we please also test for whether topics is an array, as expected? Arrays are technically a type of object, so how can we check that it's an array specifically, and not an object?